contents/node/nodejs.md

```diff
@@ -151,45 +172,44 @@ For example, inside your `package.json`, you might have
+Running `npm run build` in the command shell will execute `node app.js` and similarly `npm run lint-fix` will fix your linting errors in your JavaScript files.
+
+Apart from acting as <tooltip content="See here for some common use cases of npm scripts!">[shortcuts](https://css-tricks.com/why-npm-scripts/)</tooltip> to commonly used commands, this also sets up and encourages a consistent development and build workflow in the project.
 
 ### Benefit 7: Easy to Reuse Code From Others
 
 Node module system allows you to include other Javascript files and thus makes it easy to reuse external libraries and organize your code into separate parts with limited responsibilities.
 
 Node comes bundled with useful [core modules](https://nodejs.org/api/) such as the `fs` (file system) module which includes classes, methods and events to deal with file I/O operations and the `http` module which helps Node to transfer data over HTTP.
+There are also many useful and well-tested modules maintained by the community and external developers such as <tooltip content="A library providing various useful wrapper functions over javascript's promises">*Bluebird*</tooltip> on <tooltip content="the largest ecosystem of open source libraries in the world!">npm</tooltip>.
 
-There are also useful and well-tested non-core modules maintained by the community and external developers such as *Bluebird* for promises. Before importing them, remember to install them and include them in your `package.json` dependencies by following the instructions from the [npm section](#benefit-5-easy-dependency-management-with-npm).
+This makes development a breeze -- if a specific functionality has a large development overhead, you could look to such modules to speed things up.
 
-Importing modules is simple - just use the `require()` function and provide the module identifier or the file path.
+Importing modules is simple - simply use the `require()` function and provide the module identifier or the file path.
 ```js
 const http = require('http'); // import a core module
 const Promise = require('bluebird'); // import a non-core module
 ```
 
-Node will first check if the module identifier passed to `require` is a core module or a relative path. If so, it will return the core module or the object which references the value of `module.exports` in the specified file path respectively. Otherwise, Node will attempt to load the module from the `node_modules` folder in the parent directory of the current module.
-
-### Benefit 8: Relatively Large Developer Community
+Node will first check if the module identifier passed to `require` is a core module or a relative path. If so, it will return the core module or the value of `module.exports` in the specified file path's code. Otherwise, Node will attempt to load the module from the `node_modules` folder in the parent directory of the current module.
```

Comment:
I would phrase it the following way instead (changes in strike through and bold): 

Node will first check if the module identifier passed to `require` is a core module ~or a relative path~. If so, it will return the core module or the value of `module.exports` in the specified file path's code. Otherwise, Node will **treat the identifier as a relative path and** attempt to load the module from the `node_modules` folder in the parent directory of the current module.